src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -117,6 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CompletedProcessingLogFileFormat" xml:space="preserve">
+    <value>Processoing of SARIF log {0} is complete.</value>
```

Comment:
>{0} [](start = 36, length = 3)

We single-enquote string-valued parameters (`'{0}'`). See `ConfirmDownloadDialog_CheckboxLabel` immediately below.

Reply:
Below as well.

---
In reply to: [494522099](https://github.com/microsoft/sarif-visualstudio-extension/pull/202#discussion_r494522099) [](ancestors = 494522099)

Reply:
"Done processing SARIF log '{0}'."

---
In reply to: [494522754](https://github.com/microsoft/sarif-visualstudio-extension/pull/202#discussion_r494522754) [](ancestors = 494522754,494522099)